designs/jane/src/body.mjs

```diff
@@ -0,0 +1,219 @@
+      points.topLeft,
+      measurements.shoulderToShoulder / 2
+    )
+    points.rightShoulder = points.middle.shiftTowards(
+      points.topRight,
+      measurements.shoulderToShoulder / 2
+    )
+
+    paths.body = new Path()
+      .move(points.topRight)
+      .line(points.topLeft)
+      .line(points.bottomLeft)
+      .line(points.bottomRight)
+      .line(points.topRight)
+      .addClass('fabric')
+      .close()
+
+    paths.leftGore = new Path()
+      .move(points.leftShoulder)
+      .line(points.gorestartLeft)
+      .addClass('fabric dashed')
+      .close()
+
+    paths.rightGore = new Path()
+      .move(points.rightShoulder)
+      .line(points.gorestartRight)
+      .addClass('fabric dashed')
+      .close()
+
+    //neckline
+
+    const neckWidth = measurements.neck * options.neckWidth
+
+    points.neckLeft = points.middle.shiftTowards(points.leftShoulder, neckWidth / 2)
+    points.neckRight = points.middle.shiftTowards(points.rightShoulder, neckWidth / 2)
+
+    points.middleHem = points.bottomLeft.shiftTowards(points.bottomRight, widthBody / 2)
+
+    points.neckFront = points.middle
+      .shiftTowards(points.middleHem, measurements.hpsToBust * options.neckDepthFront)
+      .addText(`Front Neckline`, 'center')
+
+    points.neckCp1 = points.neckRight.shift(90, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckCp2 = points.neckFront.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+
+    points.neckCp3 = points.neckLeft.shift(90, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckCp4 = points.neckFront.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+
+    paths.neckLineFront = new Path()
+      .move(points.neckRight)
+      .curve(points.neckCp1, points.neckCp2, points.neckFront)
+      .curve(points.neckCp4, points.neckCp3, points.neckLeft)
+      .addClass('fabric')
+
+    points.neckBack = points.middle
+      .shiftTowards(points.neckFront, measurements.hpsToBust * options.neckDepthBack)
+      .addText(`Back Neckline`, 'center')
+
+    points.neckBackCp1 = points.neckBack.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckBackCp2 = points.neckBack.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckBackCp3 = points.neckLeft.shift(90, points.neckBack.dy(points.neckLeft) * 0.8)
+    points.neckBackCp4 = points.neckRight.shift(90, points.neckBack.dy(points.neckRight) * 0.8)
+
+    paths.necklineBack = new Path()
+      .move(points.neckRight)
+      .curve(points.neckBackCp4, points.neckBackCp2, points.neckBack)
+      .curve(points.neckBackCp1, points.neckBackCp3, points.neckLeft)
+      .addClass('fabric')
+
+    points.topTopLeft = points.topLeft.shift(90, 20)
+    points.topTopRight = points.topRight.shift(90, 20)
+    paths.extraBoundary = new Path()
+      .move(points.topLeft)
+      .move(points.topRight)
+      .move(points.topTopRight)
+      .move(points.topTopLeft)
+      .move(points.topLeft)
+      .close()
```

Comment:
```suggestion
      .move(points.topLeft)
      .addClass('fabric hidden')
      .close()
```
Need to double check if it still works when hidden like this though so best to check in gitpod.